example/user-guides/alerting/alertmanager-config-example.yaml

```
@@ -5,13 +5,52 @@ metadata:
   labels:
     alertmanagerConfig: example
 spec:
+  inhibitRules:
+  # critical > warning > info
+  #
+  # If an alert with severity 'warning'
+  # and an alert with severity 'info' occur at the same time,
+  # the alerts with severity 'info' will be ignored.
+  - sourceMatch:
+    - name: severity
+      value: critical
+    targetMatch:
+      - name: severity
+        value: warning
+      - name: severity
+        value: info
```

(nit) consistent indentation
```suggestion
    targetMatch:
    - name: severity
      value: warning
    - name: severity
      value: info
```